Secret-agency-web/src/main/react/secret-agency/src/App.css

```diff
@@ -0,0 +1,28 @@
+.App {
```

Comment:
Tohle pro nás asi v tomhle stavu nebude moc užitečný. 😄

Reply:
tak jako i to App.tsx je nam k nicemu, ale jako example usage, dokud tam nemame co dat je to podle me v pohode